src/components/Map.tsx

```diff
@@ -0,0 +1,81 @@
+import {
```

Comment:
indentation is messed up. make sure each tab is 2 spaces. (see docs/CodingStyle.md)

Reply:
done

Reply:
not all indentations are fixed.